Meadowlark-js/packages/metaed-plugin-edfi-meadowlark/src/enhancer/JoiSchemaEnhancer.ts

```diff
@@ -21,6 +21,14 @@ import { singularize, topLevelNameOnEntity } from '../Utility';
 
 const enhancerName = 'JoiSchemaEnhancer';
 
+// All descriptor documents have the same schema
+const descriptorSchema = Joi.object({
+  namespace: Joi.string().required(),
+  codeValue: Joi.string().required(),
+  shortDescription: Joi.string().required(),
+  description: Joi.string().optional(),
+});
+
```

Comment:
If we want to expand the valid fields on descriptor documents, this is the place to do it.